### PR TITLE
Link libcurl from cvmfs. Solve #5528

### PR DESCRIPTION
**Pull request by todor-ivanov**

### CODE REVIEW

**Comment by belforte** [comment]
two comments:

1. instead of parsing /etc/redhat-release  it is better to use the distribution independent way which also allows for simpler code
```
belforte@lxplus618/~> lsb_release -rs
6.10
```
e.g. `os_ver=$(lsb_release -rs|cut -d. -f1)` 

2. let's take this chance to get rid of `/opt/glidecondor` and avoid putting `'.'` twice in PATH and LD_LIBRARY_PATH !

**Comment by todor-ivanov** [comment]
Totally agree with you.
And then we should also get rid of:
export PYTHONPATH=/opt/glidecondor/lib/python:$PYTHONPATH 
(adding the new commit to the current PR now)

**Comment by bbockelm** [comment]
Note that `lsb_release` is not part of the requirements for bootstrapping CMS software -- I'd be a bit reluctant to assume it is everywhere (pulls in an enormous number of dependencies).

Going forward, most distros are switching to `os-release`; see https://www.freedesktop.org/software/systemd/man/os-release.html.  To add headache, that started after RHEL6...

Fine to use it if it is there, of course.

**Comment by belforte** [comment]
thanks @bbockelm but we only need this in the schedd's . Not everywhere CMSW runs !
fine to go for `os-release` . thanks for the tip !

**Comment by todor-ivanov** [comment]
@bbockelm thanks for the tip - I didn't  think about that before. I agree that /etc/os-release provides a nice well defined and standardised  way of getting the Os version, but unfortunately it is missing in the slc6 machines. In our case it was needed a way compatible with both cc7 and slc6.

**Comment by belforte** [comment]
@todor-ivanov since you call the init.sh in curl_path, that already takes care of PATH and LD_LIBRARY_PATH so you should not touch those later in the code. Simply call that after you defined the path for glidecondor

**Comment by todor-ivanov** [comment]
@belforte you are perfectly correct. I added another commit to the current PR - the additional export was removed and the order rearranged.

File: scripts/dag_bootstrap.sh [hunk 1]
```diff
@@ -82,13 +82,22 @@ then
...
+os_ver=$(lsb_release -rs|cut -d. -f1)
+curl_path="/cvmfs/cms.cern.ch/slc${os_ver}_amd64_gcc700/external/curl/7.59.0"
+libcurl_path="${curl_path}/lib"
+source ${curl_path}/etc/profile.d/init.sh
+
 export PYTHONUNBUFFERED=1
 echo "Printing current environment..."
+
+srcname=$0
+env > ${srcname%.sh}.env
```
Comment (belforte) [inline scripts/dag_bootstrap.sh]: what is the purpose of this line ?

**Comment by belforte** [comment]
I have one last question, but am merging anyhow so we can put in dev/pre-prod TW and test. We can resolve that question independenlty

**Pull request merged**